src/sentry/static/sentry/app/views/organizationStats/index.jsx

```diff
@@ -146,11 +153,13 @@ const OrganizationStats = createReactClass({
           group: 'project',
         },
         success: data => {
-          this.state.rawProjectData[statName] = data;
-          this.state.projectsRequestsPending -= 1;
-          this.setState({
-            rawProjectData: this.state.rawProjectData,
-            projectsRequestsPending: this.state.projectsRequestsPending,
+          let rawProjectData = this.state;
+          rawProjectData[statName] = data;
+          this.setState(state => {
+            return {
+              rawProjectData,
+              projectsRequestsPending: this.state.projectsRequestsPending - 1,
```

Comment:
`this.state` --> `state`

Reply:
this is a good reason to use "prevState" as the name. fixed a few bugs in this code thanks to the integration test

Reply:
it's not really previous state though, it's more current state

Reply:
... true ... but it's close enough right? willing to change it if you have an idea